src/plugins/embeddable/public/embeddable_panel/embeddable_loading_indicator.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiLoadingChart, EuiPanel } from '@elastic/eui';
+
+export const EmbeddableLoadingIndicator = ({ showShadow }: { showShadow?: boolean }) => {
```

Comment:
Is showShadow prop needed? Its never used.

Reply:
It's used once here `x-pack/plugins/lens/public/embeddable/embeddable_component.tsx`.